app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,90 @@
+class TasksController < ApplicationController
+  TASKS = [
+    { name: "cleaning", description: "vacuum" },
+    { name: "dinner", description: "make soup" },
+    { name: "shopping", description: "Go to Target" },
+  ]
```

Comment:
Don't forget to refactor your code often, so you remember to delete unused variables like this!